package.json

```diff
@@ -129,12 +129,12 @@
     "@angular/cli": "^14.2.3",
     "@angular/compiler-cli": "^14.2.3",
     "@angular/language-service": "^14.2.3",
+    "@babel/runtime": "^7.18.9",
     "@ngrx/store-devtools": "^14.3.0",
```

Comment:
not related to this file, but remove the references of changelog adaptation changes from `creating-lib.md`. Starting from line 270